api/v1alpha1/schedule_types.go

```diff
@@ -104,3 +109,17 @@ func (s *Schedule) GetStatus() *Status {
 func (s *Schedule) GetResources() corev1.ResourceRequirements {
 	return s.Spec.ResourceRequirementsTemplate
 }
+
+// IsNonStandard returns true if the value begins with "@",
+// indicating a special definition.
+// An example is '@weekly-random'
```

Comment:
I've chosen the term _non-standard_ because it corresponds to the documentation at https://k8up.io/k8up/1.0.0/references/schedule-specification.html.

Reply:
I understand where your reasoning comes from. I think it makes sense to open a PR so that we separate between non-standard and k8up-specific schedules

Reply:
there you go: https://github.com/vshn/k8up/pull/267